### ISSUE TITLE
menu not showing on flutter web with --release flag

### ISSUE THREAD

**Issue by rinde** [icemanbsi/searchable_dropdown#80]
Consider this snippet:
```dart
SearchableDropdown.single(
  dialogBox: false,
  menuConstraints: BoxConstraints.tight(Size.fromHeight(350)),
  /* rest of parameters are hidden for brevity */
),
```

It works as expected when running with: `flutter run -d chrome`

When running with: `flutter run --release -d chrome` the menu is not expanded when clicking on the dropdown. The mouse click does seem to register as the InkWell is displayed as well as the highlight on mouse hover. When choosing `dialogBox: true` the code also works as expected, this could be a good workaround if the dialogBox wasn't screen filling on web.

I've tested this on the following flutter versions (all with the same result):
- 1.20.0-2.0.pre (dev)
- 1.19.0-4.2.pre (beta)
- 1.16.3

I'm not sure if the problem lies with flutter itself or if it is fixable in this project.

**Comment by hoonblizz** [comment]
I had the same issue. 
It works good on debug mode but not in release mode. 
Thankfully `dialogBox: true` made it functional in release mode. (Of course it's against my UI but it works)

**Comment by Aerofluxx** [comment]
I can still confirm this behavior.

- Tested on 1.25.0-8.3.pre (beta)

Build with: `flutter build web --release --web-renderer canvaskit` and also tried `flutter build web --release --web-renderer html`

As above `dialogBox: true` works but it's just not user friendly to push a huge screen above the whole content.

**Comment by maxime915** [comment]
I can still confirm this behavior.

- Tested on Flutter 1.26.0-17.6.pre (beta)

Same build options as @Aerofluxx . I can also confirm that changing `dialogBox: true` works with both renderer.

EDIT: `flutter run --profile` doesn't work either

**Comment by lcuis** [comment]
Can you please try with the `search_choices` plugin to know whether there is a difference between the 2 plugins on this point? So far, I never had issues with running in release mode on web, android, ios and linux.

**Comment by maxime915** [comment]
> Can you please try with the `search_choices` plugin to know whether there is a difference between the 2 plugins on this point? So far, I never had issues with running in release mode on web, android, ios and linux.

Yes it works perfectly with the `search_choices` plugin in release mode on web (with both flutter run and flutter build), this may solve our issue, thanks!

**Comment by lcuis** [comment]
> > Can you please try with the `search_choices` plugin to know whether there is a difference between the 2 plugins on this point? So far, I never had issues with running in release mode on web, android, ios and linux.
> 
> Yes it works perfectly with the `search_choices` plugin in release mode on web (with both flutter run and flutter build), this may solve our issue, thanks!

Thanks for letting us know @maxime915 .
It would be good to understand where the discrepancy comes from and what exactly causes this failure for the `searchable_dropdown` that doesn’t affect `search_choices`.
